site/docs-md/guides/ionic-framework-app.md

```diff
@@ -0,0 +1,250 @@
+
+```shell
+npx cap sync
+```
+
+Next, run the Capacitor `open` command, which opens the native iOS project in Xcode:
+
+```shell
+npx cap open ios
+```
+
+Within Xcode, click on `App` in the Project Navigator on the left-hand side, then within the `Signing` section, select your Development Team. 
+
+![Xcode - Selecting Development Team](/assets/img/docs/guides/ionic-framework/xcode-signing.png)
+
+Next, for the Camera plugin to work, we must configure the "Privacy - Camera Usage" permission. iOS displays a modal dialog automatically after the first time that `Camera.getPhoto()` is called, prompting the user to allow the app to use the Camera. To set this, the `Info.plist` file must be modified ([more details here](../ios/configuration/)). To access it, click "Info," then expand "Custom iOS Target Properties."
+
+![Xcode Custom iOS Target Properties](/assets/img/docs/guides/ionic-framework/xcode-info-plist.png)
+
+Each setting in `Info.plist` has a low-level parameter name and a high-level name. By default, the property list editor shows the high-level names, but it's often useful to switch to showing the raw, low-level names. To do this, right-click anywhere in the property list editor and toggle "Show Raw Keys/Values."
+
+Locate the `NSCameraUsageDescription` Key (or add it) and set the Value to something that describes why the app needs to use the camera, such as "To Take Photos." The Value will be displayed to the app user when the permission prompt opens.
+
+With permissions in place, we are ready to try out the app on a real device! Connect an iOS device to your Mac computer, then within Xcode click the "Build" button to build, install, and launch the app on your device:
+
+![Xcode build button](/assets/img/docs/guides/ionic-framework/xcode-build-button.png)
+
+Upon tapping the Camera button on Tab Two, the permission prompt will display. Tap OK, then take a picture with the Camera. Afterward, the photo shows in the app:
+
+![iOS Camera permissions](/assets/img/docs/guides/ionic-framework/ios-permissions-photo.png)
+
+## Android
+Capacitor Android apps are configured and managed through Android Studio. Before running this app on an Android device, there's a couple of steps to complete.
+
+From the Terminal, run the Capacitor `sync` command, which updates all native code dependencies as well as copies all web assets (the Ionic Angular app in this case) to the native Android project:
+
+```shell
+npx cap sync
+```
+
+Next, run the Capacitor `open` command, which opens the native Android project in Android Studio:
+
+```shell
+npx cap open android
+```
+
+Similar to iOS, we must enable the correct permissions to use the Camera. Configure these in the `AndroidManifest.xml` file. Android Studio will likely open this file automatically, but in case it doesn't, locate it under `android/app/src/main/`.
```

Comment:
Same as on iOS, Capacitor already includes all the required permissions, and in this case user don't even have to configure a text, so not sure if it's worth commenting.